src/components/post.js

```diff
@@ -43,6 +43,19 @@ const Post = ({ post, relatedPost }) => {
     ? `https://github.com/${author.github}.png?size=100`
     : `https://ui-avatars.com/api/?name=${author.id}&size=460`
 
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      const mainDoc = document.querySelectorAll(".container-post a")
```

Comment:
Hi @pompodars Directly interacting with DOM elements is not recommended in react, I think we should use some better approach to handle this.
For eg : https://www.surinderbhomra.com/Blog/2020/05/19/GatsbyJS-Markdown-Plugin-Open-External-Links-New-Tab